azure-pipelines.yml

```diff
@@ -214,14 +205,13 @@ jobs:
       #   shallow exclusion; tests deeper under //hw will still be found.
       # * It excludes targets that depend on bitstream_splice rules, since the
       #   environment does not have access to Vivado.
-      export GCP_BAZEL_CACHE_KEY=$(bazelCacheGcpKeyPath)
       TARGET_PATTERN_FILE=target_pattern.txt
       echo //... > "${TARGET_PATTERN_FILE}"
       echo -//quality/... >> "${TARGET_PATTERN_FILE}"
       echo -//sw/otbn/crypto/... >> "${TARGET_PATTERN_FILE}"
       echo -//third_party/riscv-compliance/... >> "${TARGET_PATTERN_FILE}"
       echo -//hw:all >> "${TARGET_PATTERN_FILE}"
-      ./bazelisk.sh cquery \
+      ci/bazelisk.sh cquery \
```

Comment:
Since this is just a query, I am not sure that it makes a difference whether it uses a the cache or not. Also, if I recall correctly, `ci/bazellisk.sh` can print things that could make the output unusuable for queries (I might recall incorrectly).

Reply:
You're right about it printing output when it shouldn't - I've moved those to `stderr` so they shouldn't interfere.

Do you think there's much of a penalty of using `ci/bazelisk.sh` for everything in CI? It might be easier to be consistent and use it for everything. I don't think everybody knows you need to use that script to get caching, so using it consistently in CI may encourage others to use it when adding things?

Reply:
Oh, I do see that our custom `bazelisk.sh` commands (like outquery) aren't working when run through `ci/bazelisk.sh`, but I'm not sure why :thinking:

Maybe I will move those back to `./bazelisk.sh` if I can't work out what's wrong.

Reply:
I tend to agree that using `ci/bazelisk.sh` consistently would be better. This script calls into `bazelisk.sh` so in principle our custom commands should work?

Reply:
Ah, it's because `./bazelisk.sh` only checks `$1` for the command (e.g. `outquery`) while `ci/bazelisk.sh` adds some flags before your arguments (i.e. `--bazelrc=/tmp/...`).

We could change the order that `ci/bazelisk.sh` passes its flags or give `./bazelisk.sh` better argument parsing.

Reply:
I've made `./bazelisk.sh` correctly handle flags that have to come before the subcommand (like `--bazelrc=...`) but it's using some fairly ugly Bash. It's valid bash AFAICT (shellcheck is okay with it) but let me know if you think it's acceptable.